schemas/gen-missing-crd-schemas.sh

```diff
@@ -3,27 +3,28 @@
 # run from root
 
 # this script will create a tar.gz with missing crds, for consumption by kubeval (validate-templates)
-cd schemas/ || exit
+cd schemas/
 
 export FILENAME_FORMAT='{kind}-{group}-{version}'
 
 input_folder="input-crds"
 gen_folder="generated-crd-schemas"
 
 mkdir $gen_folder >/dev/null
-rm -rf ./$gen_folder/*
+#rm -rf ./$gen_folder/*
 mkdir $input_folder >/dev/null
 rm -rf $input_folder/*
 
 # match all the crds of charts that didn't ship crds (some operators don't), and pull them
 # (expects kube context to have a cluster that has them all)
-for pkg in "argoproj" "cert-manager" "istio" "jaeger" "kiali" "knative"; do
+# for pkg in "argoproj" "operators.coreos" "cert-manager" "istio" "jaeger" "kiali" "knative"; do
+for pkg in "operators.coreos"; do
```

Comment:
Does it mean that we do not need to generate CRDs for `argoproj" "cert-manager" "istio" "jaeger" "kiali" "knative";` anymore ?

Reply:
no, this is a manual script, and the one using it should know what to do. If not all of those are deployed, but only one is, and that one needs to be added, this is the way to do it.